input/pagecontent/terminology.md

```diff
@@ -61,11 +69,15 @@ The list below shows the value sets bound to a supported element or element slic
 |[PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-pbs-item.html)|[AU Core MedicationRequest](StructureDefinition-au-core-medicationrequest.html), [AU Core Medication](StructureDefinition-au-core-medication.html)|AU Base|
 |[Practitioner Role](https://healthterminologies.gov.au/fhir/ValueSet/practitioner-role-1)|[AU Core PractitionerRole](StructureDefinition-au-core-practitionerrole.html)|NCTS|
 |[Procedure](https://healthterminologies.gov.au/fhir/ValueSet/procedure-1)|[AU Core Procedure](StructureDefinition-au-core-procedure.html)|NCTS|
+|[QuantityComparator](http://hl7.org/fhir/ValueSet/quantity-comparator)|[AU Core Blood Pressure](StructureDefinition-au-core-bloodpressure.html), [AU Core Body Height](StructureDefinition-au-core-heartrate.html), [AU Core Body Temperature](StructureDefinition-au-core-bodytemp.html), [AU Core Body Weight](StructureDefinition-au-core-bodyweight.html), [AU Core Diagnostic Result Observation](StructureDefinition-au-core-diagnosticresult.html), [AU Core Heart Rate](StructureDefinition-au-core-heartrate.html), [AU Core Pathology Result Observation](StructureDefinition-au-core-diagnosticresult-path.html), [AU Core Respiration Rate](StructureDefinition-au-core-resprate.html), [AU Core Waist Circumference](StructureDefinition-au-core-waistcircum.html)|FHIR|
```

Comment:
Amend AU Core Body Height link from heartrate to body height: [AU Core Body Height](StructureDefinition-au-core-heartrate.html)

Reply:
fixed